resources/shaders/rt_hybrid.hlsl

```diff
@@ -277,13 +278,16 @@ void ReflectionHit(inout ReflectionHitInfo payload, in MyAttributes attr)
     float3 kD = 1.0 - kS;
     kD *= 1.0 - metal;
 
+	const float2 sampled_brdf = brdf_lut.SampleLevel(s0, float2(max(dot(fN, V), 0.01f), roughness), 0).rg;
+
 	//Lighting
 	float3 lighting = shade_pixel(hit_pos, V, albedo, metal, roughness, fN, payload.seed, payload.depth);
 
 	//Reflection in reflections
 	float3 reflection = DoReflection(hit_pos, V, fN, payload.seed, payload.depth + 1);
 
 	float3 specular = reflection * F;
+	//float3 specular = reflection * (kS * sampled_brdf.x + sampled_brdf.y);
```

Comment:
We need to remember to uncomment this, therefore I added a comment here 👌

Reply:
donzo

Reply:
donzo